templates/common/infra/bicep/core/ai/cognitiveservices.bicep

```diff
@@ -23,13 +23,16 @@ resource account 'Microsoft.CognitiveServices/accounts@2022-10-01' = {
 }
 
 @batchSize(1)
-resource deployment 'Microsoft.CognitiveServices/accounts/deployments@2022-10-01' = [for deployment in deployments: {
+resource deployment 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01' = [for deployment in deployments: {
   parent: account
   name: deployment.name
   properties: {
     model: deployment.model
     raiPolicyName: contains(deployment, 'raiPolicyName') ? deployment.raiPolicyName : null
-    scaleSettings: deployment.scaleSettings
```

Comment:
It looks like scaleSettings is still a supported property on the resource - do we no longer want to copy the value as we had done before?

Reply:
I have confirmed with the team that scaleSettings will be removed in the next version and they recommend not supporting it now.  I will keep it out.